en/components/grid.md

```diff
@@ -543,6 +543,144 @@ public ngOnInit() {
     ];
 }
 ```
+<div class="divider--half"></div>
+
+### Summaries
+> [!NOTE]
+> The summary of the column on is a **function of all column values**, even if paging, filtering or sorting is applied.
+ 
+Grid **summaries** can also be enabled per-column level, which means that you can activate it only for columns that you need to. **Grid summaries** gives you a predefined set of default summaries, depending on the type of data in the column, so that you can save some time:
+
+For `string` and `boolean` data type is available the following function:
+ - count
+
+For `number` data type are available the following functions:
+ - count
+ - min
+ - max
+ - average
+ - sum
+
+For `date` data type are available the following functions:
+ - count
+ - earliest
+ - latest
+
+**Grid summaries** are enabled per-column by setting `hasSummary` property to `true`. It is also important to keep in mind that the summaries for each column are resolved accordind column data type. In the `igx-grid` default column data type is `string`, so if you want `number` or `date` specific summaries you should specify `dataType` property to `number` or `date`.
```

Comment:
according to the column data type.  In the `igx-grid`, the default...
... you should specify the `dataType` property as `number` or `date`.
You can use "set" instead of "specify", and then you can keep "to" instead of "as".